modules/ROOT/pages/introduction/anypoint-connector-authentication.adoc

```diff
@@ -0,0 +1,23 @@
+= Introduction to Anypoint Connectors
```

Comment:
General comments: 

Do we want to distinguish between OAuth 1.0 and OAuth 2.0 or at least mention them? I think some connectors still support OAuth 1.0. Also, some connectors call Basic authentication User and Password authentication. I will try to verify this.

I wonder if this information should go into the Configuration Overview topic that I just sent to Kim for editing. I mention authentication there when I talk about the global element.